.gitignore

```diff
@@ -35,3 +35,6 @@ jspm_packages
 
 # Optional REPL history
 .node_repl_history
+
+lib
```

Comment:
ah, I'll make the description more clear - the user doesn't run the build task here, it's just defined so that `prepublish` can run it automatically during `npm publish` so that the version that is uploaded to `npm` _does_ have the built version in it. Git will keep track of the source code while npm will contain the built CommonJS-compatible code, similar to how React and some other projects are set up.